lib/cartopy/tests/crs/test_oblique_mercator.py

```diff
@@ -0,0 +1,193 @@
+        ParamTuple(
+            "default",
+            dict(),
+            dict(),
+            (-245106.75804, 5626768.52447),
+            (35451.51708, 5595849.69689),
+        ),
+        ParamTuple(
+            "azimuth",
+            dict(azimuth=90.0),
+            dict(alpha="89.999"),
+            (-386712.17018, 6540102.97351),
+            (55680.57266, 6500330.56121),
+        ),
+        ParamTuple(
+            "central_longitude",
+            dict(central_longitude=90.0),
+            dict(lonc="90.0"),
+            (-4739202.85619, 10329047.01897),
+            (-4786583.034, 9966930.01085),
+        ),
+        ParamTuple(
+            "central_latitude",
+            dict(central_latitude=45.0),
+            dict(lat_0="45.0"),
+            (-245269.04118, 642564.31415),
+            (35474.56405, 611638.73957),
+        ),
+        ParamTuple(
+            "false_easting_northing",
+            dict(false_easting=1000000, false_northing=-2000000),
+            dict(x_0="1000000", y_0="-2000000"),
+            (754893.24196, 3626768.52447),
+            (1035451.51708, 3595849.69689),
+        ),
+        ParamTuple(
+            "scale_factor",
+            # Number inherited from test_mercator.py .
+            dict(scale_factor=0.939692620786),
+            dict(k="0.939692620786"),
+            (-230325.01183, 5287432.86131),
+            (33313.52899, 5258378.6672),
+        ),
+        ParamTuple(
+            "globe",
+            dict(globe=ccrs.Globe(ellipse="sphere")),
+            dict(ellps="sphere"),
+            (-244502.86059, 5646357.44304),
+            (35364.23322, 5615460.21872),
+        ),
+        ParamTuple(
+            "combo",
+            dict(
+                azimuth=90.0,
+                central_longitude=90.0,
+                central_latitude=45.0,
+                false_easting=1000000,
+                false_northing=-2000000,
+                scale_factor=0.939692620786,
+                globe=ccrs.Globe(ellipse="sphere"),
+            ),
+            dict(
+                alpha="89.999",
+                lonc="90.0",
+                lat_0="45.0",
+                x_0="1000000",
+                y_0="-2000000",
+                k="0.939692620786",
+                ellps="sphere",
+            ),
+            (-4279982.08123, 1916861.68937),
+            (-4138080.80706, 1631302.04295),
+        ),
+    ]
+    param_ids: List[str] = [p.id for p in param_list]
+
+    @pytest.fixture(autouse=True, params=param_list, ids=param_ids)
+    def make_variant_inputs(self, request) -> None:
+        inputs: TestCrsArgs.ParamTuple = request.param
+
+        self.oblique_mercator = ccrs.ObliqueMercator(**inputs.crs_kwargs)
+        proj_kwargs_expected = dict(
+            ellps="WGS84",
+            lonc="0.0",
+            lat_0="0.0",
+            k="1.0",
+            x_0="0.0",
+            y_0="0.0",
+            alpha="0.0",
+            units="m",
+        )
```

Comment:
@trexfeathers Minor, you could move this to be a class attribute (akin to `TestCrsArgs.point_a_plate_carree` and `TestCrsArgs.point_b_plate_carree`) and save the exact same `dict` from being recreated each call to this fixture.

Reply:
7ec343b9dade3571a8ce90d482cbf3d533e2aec9